content/CORE/CORETutorials/SystemConfiguration/CreatingCAsandCertificates/CreatingCertificateAuthorities.md

```diff
@@ -11,129 +11,109 @@ tags:
 
 {{< toc >}}
 
-TrueNAS can act as a Certificate Authority (CA). When encrypting SSL or TLS connections to the TrueNAS system, you can import an existing CA or create a CA and certificate on the TrueNAS system. The certificate will appear in the drop-down menus for services that support SSL or TLS. 
+TrueNAS can act as a certificate authority (CA). 
+When encrypting SSL or TLS connections to the TrueNAS system, you can import an existing CA or create a CA and certificate on the TrueNAS system. 
+The certificate appears on the dropdown menus for services that support SSL or TLS. 
 
-Go to **System > CAs** and click **ADD**. Name the CA, then choose the **Type**. The three type options are *Internal CA*, *Intermediate CA*, and *Import CA*. The process for each type is slightly different.
+Go to **System > CAs** and click **ADD**. Enter a name for the CA, then choose the type from the **Type** dropdown list of three, **Internal CA**, **Intermediate CA**, or **Import CA**. The process to add a CA for each type is slightly different.
 
-## Internal CA
+## Creating CA
 
-### Identifier and Type
+A CA must exist in CORE to add an Intermediate CA. This can be an internal or imported CA.
 
-Set *Internal CA* as the **Type**. 
+To create a CA:
 
-You can select a profile for the CA to auto-fill options like **Key Type**, **Key Length**, **Digest Algorithm**. Otherwise, you must set options manually.
+1. Enter or select the **Identifier and Type** setting options.
+   
+   ![AddInternalCAIdentTypeNoProfile](/images/CORE/13.0/AddInternalCAIdentTypeNoProfile.png "Add Internal CA Name and Type")
 
-![Internal CA Identifier Type](/images/CORE/12.0/InternalCAIdentifierType.png)
+   a. Enter a name for this CA.
+   b. Select **Internal CA** from the **Type** dropdown list to create an internal certificate. 
+      Select **Intermediate CA** to create an intermediate certificate. This displays the **Signing Certificate Authority** field in **Certificate Options**.
 
-### Certificate Options
+2. Select an option from the **Profiles** dropdown list. 
+   A profile for the CA auto-fills options like **Key Type**, **Key Length**, **Digest Algorithm**. Otherwise, you must set options manually.
```

Comment:
```suggestion
2. Select an option from the **Profiles** dropdown list. 
   A profile for the CA auto-fills options like **Key Type**, **Key Length**, and **Digest Algorithm**. Otherwise, you must set options manually.
```